### DISCUSSION TITLE
Agile price update notification

### DISCUSSION THREAD

**Discussion by gorstj** [BottlecapDave/HomeAssistant-OctopusEnergy#335] [Ideas]
Is it possible to have an entity update when the next block of agile price updates occur?

Essentially I want to automate some functions (e.g. a notification) when the pricing gets released?

Any ideas?

**Comment by BottlecapDave** [comment]
Hello. The current price sensor will update within a few minutes of new prices being downloaded, which will update the `rates` attribute on the sensor. You could probably create an automation which triggers when this [attribute changes](https://www.home-assistant.io/docs/automation/trigger/#triggering-on-attribute-changes)
